### DISCUSSION TITLE
Boas Vindas

### DISCUSSION THREAD

**Discussion by EuricoDNJR** [EuricoDNJR/SG-VILLA-DOLCE#56] [Announcements]
Pessoal, acabei encontrando essa funcionalidade do GitHub, onde permite a gente interagir de modo a criar e responder dúvidas por aqui mesmo, ao invés de utilizar até mesmo alguns tipos de comentários dos Cards.

**Comment by lucassl2020** [comment]
Show dms! Obrigado pelas boas-vindas chefe, vc é o melhor!
